bitnami/minio/README.md

```diff
@@ -95,7 +95,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `auth.forcePassword`       | Force users to specify required passwords                                                                                                                                                                 | `false`                        |
 | `auth.useCredentialsFiles` | Mount credentials as a files instead of using an environment variable                                                                                                                                     | `false`                        |
 | `auth.forceNewKeys`        | Force root credentials (user and password) to be reconfigured every time they change in the secrets                                                                                                       | `false`                        |
-| `defaultBuckets`           | Comma, semi-colon or space separated list of buckets to create at initialization (only in standalone mode)                                                                                                | `""`                           |
+| `defaultBuckets`           | Comma, semi-colon or space separated list of buckets to create at initialization (only in standalone mode). Do not set this value in distributed mode, as it will cause MinIO&reg; to crash.                                                                                                | `""`                           |
```

Comment:
Please note this table is automatically updated using the metadata from the values.yaml, see https://github.com/bitnami/charts/blob/main/bitnami/minio/values.yaml#L119.

You should perform the change there and the table will be automatically updated by https://github.com/bitnami/readme-generator-for-helm

Reply:
Okay, I've fixed it